common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -129,6 +129,7 @@
                     head = document.getElementsByTagName('head')[0];
                     s.innerHTML = c;
                     s.setAttribute('data-loaded-from', 'local');
+                    s.className = "gu-loggable";
```

Comment:
Will most likely have a merge conflict with #8205 so make sure to get @sndrs to add it to the new link elements.